src/rust/Cargo.toml

```diff
@@ -35,7 +35,7 @@ toml = "=0.7.8"
 version = "=20.1.0"
 git = "https://github.com/stellar/rs-soroban-env"
 package = "soroban-env-host"
-rev = "c1b238b65bfd13666be4ac14e0e390c31b549caf"
+rev = "c2fae7dafe630358754eb9f3d235abfd7f8a3b08"
```

Comment:
I had to update the `soroban-env-host` ref as well because cargo deny complains about duplicates if you only update the `soroban-test-wasms` ref. We should fix this, but for now this is fine because the new changes don't affect the protocol.